{{cookiecutter.repo_name}}/templates/mdimechanic_python.yml

```diff
@@ -0,0 +1,19 @@
+code_name: '{{ cookiecutter.repo_name }}'
+docker:
+  image_name: '{{ cookiecutter.repo_name }}'
+
+  build_image:
+    - ""
+
+  build_code:
+    - pip install -r {{ cookiecutter.repo_name }}/requirements.txt
```

Comment:
The `pip install` line should be moved to the `build_image` step; otherwise its changes won't be persistent.  Also, it won't be possible to use a requirements.txt file that is stored in the repo, as this cannot be volume mounted during an image build step.  The user should simply include all requirements in the `build_image` step of `mdimechanic.yml`.

Reply:
Is there a way to provide the requirement both in the case when someone is not using `mdimechanic.yml` and when someone is not? If they choose `no` for mdimechanic, we would want to inform them somehow to install `pymdi` if they're using Python. The `requirements.txt` would be a good way to do that if it could be mounted or included during the `build_image` step.